assets/javascripts/kitten/components/icons/messenger-icon.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+export const MessengerIcon = props => (
+  <svg {...props}>
+    <path
+      d="M10 .938c-5.523 0-10 4.171-10 9.317 0 2.932 1.454 5.547 3.727 7.255v3.552l3.405-1.88c.908.253 1.871.39 2.868.39 5.523 0 10-4.172 10-9.317C20 5.109 15.523.938 10 .938zm.994 12.546l-2.547-2.732-4.969 2.732 5.466-5.838 2.609 2.733 4.907-2.733-5.466 5.838z"
+      fill="#FFF"
```

Comment:
On ne donne pas la possibilité de changer la couleur de l'icône si besoin ?

Reply:
Bonne question. On ne le fait pas jusqu'à présent, mais si le besoin se présente on pourra modifier le composant ?

Reply:
Ça me va. Je changerai juste la casse en minuscule :).
```suggestion
      fill="#fff"
```